src/venue/core.cljs

```diff
@@ -72,42 +77,19 @@
   (let [current-id (:current @cursor)]
     (current-id (:fixtures cursor))))
 
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-
-(defn raise!
-  [owner event data]
-  (let [c (om/get-shared owner [:event-chan])]
-    (put! c [event data])))
+(defn- fixture-by-cursor
+  [cursor]
+  (->> @venue-state
+       (map val)
+       (mapcat :fixtures)
+       (map second)
+       (filter #(= (:state %) @cursor))
+       first))
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-
-(defn get-route
-  ([id]
-   (get-route id {}))
-  ([id opts]
-   (let [route (->> id fixture-by-id :route)]
-     (secretary/render-route route opts))))
-
-(defn navigate!
-  ([id]
-   (navigate! id {}))
-  ([id opts]
-   (let [route (get-route id opts)]
-     (if (:static route)
-       (throw (js/Error. "Cannot navigate to a static fixture!")))
-     (log-info "Navigating to " route)
-     (set! (.. js/document -location -href) route))))
-
-(defn start!
-  []
-  (when (not (:started? @state))
-    (swap! state assoc :started? true)
-    (log-info "Starting up...")
-    (secretary/dispatch! js/window.location.hash)))
-
 (defn- launch-route!
```

Comment:
So this is a late-night comment (by my standards): so far your functions have been small, well-contained units. This may obviously be something Om-specific (which I don't know ver well), but this function feels like a big clump?  Could it be split up in clearly named units?
(again, will have another look later, ignore me if I'm wrong)